.devcontainer/dev/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM registry.opensuse.org/opensuse/leap:15.5
```

Comment:
I think this would be required for installing `obs-to-maven` ```
RUN zypper addrepo --no-gpgcheck https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Utils/openSUSE_Leap_15.5 systemsmanagement:uyuni:utils
```